doc/sphinx/changes.rst

```diff
@@ -8,6 +8,967 @@ Recent changes
 
    .. include:: ../unreleased.rst
 
+Version 8.20
+------------
+
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+Coq version 8.20 TODO_main_new_features
+
+We highlight some of the most impactful changes here:
+
+  - :ref:`rewrite_rules`
+
+  - :ref:`primitive strings <pstring_library>`
+
+  - TODO other notable changes
```

Comment:
A lot of work went into reducing the size of the bytecode segment, which in turn means that `.vo` files might now be considerably smaller.